test/Microsoft.Azure.SignalR.Common.Tests/ConnectionStringParserFacts.cs

```diff
@@ -66,14 +64,13 @@ public void ValidApplicationConnectionString(string expectedEndpoint, string con
                 Assert.IsType<ClientSecretCredential>(aadAccessKey.TokenCredential);
             }
             Assert.Null(version);
-            Assert.Null(accessKey.Port);
             Assert.Null(clientEndpoint);
         }
 
         [Theory]
-        [InlineData("https://aaa", "1.0", null, "endpoint=https://aaa;AccessKey=bbb;version=1.0")]
-        [InlineData("https://aaa", "1.0-preview", null, "ENDPOINT=https://aaa/;ACCESSKEY=bbb;VERSION=1.0-preview")]
-        [InlineData("http://aaa", "1.1", null, "endpoint=http://aaa;AccessKey=bbb;Version=1.1")]
+        [InlineData("https://aaa", "1.0", 443, "endpoint=https://aaa;AccessKey=bbb;version=1.0")]
```

Comment:
add some tests for when Endpoint contains port, e.g. `Endpoint=https://aaa:5040`?